lib/smart_proxy_container_gateway/database.rb

```diff
@@ -0,0 +1,26 @@
+require 'sequel'
+
+module Proxy
+  module ContainerGateway
+    module Database
+      class << self
+        Sequel.extension :migration, :core_extensions
+      end
```

Comment:
I don't know if this is a correct implementation. Perhaps it needs to be done before the `connect` call. It also needs a `attr_reader :connection` to expose the connection.